scripts/cli/environment.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
       name: "pmesh-primary-node",
       script: "../../target/release/polymesh",
       args:
-        "-d /tmp/pmesh-primary-node --pool-limit 100000 --ws-port 9944 --rpc-cors all --alice --validator --chain " +
+        "-d /tmp/pmesh-primary-node --pruning=archive --rpc-methods=unsafe --rpc-external --ws-external --pool-limit 100000 --ws-port 9944 --rpc-cors all --alice --validator --chain " +
```

Comment:
FYI, this doesn't change the CI integration test setup.  To update the CI-pipeline we will need to change the parameters in the `concourse` repo.

The current pipeline setup uses `--unsafe-rpc-external --unsafe-ws-external` instead of `--rpc-methods=unsafe --rpc-external --ws-external`.

Also `--pruning=archive` is missing.

@raycar5 Do you think we need to change those parameters in the pipeline?

Reply:
Also our pipeline seems to be running the tests using the `*.js` files and not the `*.ts` files.  Not sure what affect that has.